app/lib/database/QueryBuilder.js

```diff
@@ -86,75 +53,102 @@ const controlForNoArrays = {
             string: [ops.NOTLIKE, ops.AND],
             number: [ops.NE, ops.AND],
         },
-        from: [ops.FROM, ops.AND],
-        to: [ops.TO, ops.AND],
+        // from: [ops.FROM, ops.OR],
+        // to: [ops.TO, ops.OR],
+        between: [ops.BETWEEN, ops.OR],
     },
 
     array: {
         match: {
-            string: ops.LIKE,
-            number: ops.EQ,
+            string: [ops.LIKE, ops.OR],
+            number: [ops.IN, ops.OR],
         },
         exclude: {
-            string: ops.NOTLIKE,
-            number: ops.NE,
+            string: [ops.NOTLIKE, ops.AND],
+            number: [ops.NOTIN, ops.AND],
         },
-        from: ops.FROM,
-        to: ops.TO,
+        // from: [ops.FROM, ops.OR],
+        // to: [ops.TO, ops.OR],
     },
 }
 
+const handleBetween = (fieldName, pairsLi) => {
```

Comment:
I don't think that `pairsLi` is a good variable name. Could you please change it?